src/core/cli/lifecycle/kill_zombie_linux.go

```
@@ -0,0 +1,46 @@
+//
+// Returns (false, nil) when /proc/<pid>/stat is missing or unreadable — the
+// caller's aliveness check is authoritative for that case. No I/O on the proc
+// filesystem maps to the transient-fork failure mode the (bool, error)
+// contract exists for, so this implementation never returns a non-nil error.
+func isZombie(pid int) (bool, error) {
+	if pid <= 0 {
+		return false, nil
+	}
+	data, err := os.ReadFile("/proc/" + strconv.Itoa(pid) + "/stat")
+	if err != nil {
+		return false, nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 31) but it returns nil

(nilerr)